Sources/StreamChat/APIClient/Endpoints/Payloads/MessageTranslationsPayload.swift

```diff
@@ -35,11 +35,11 @@ public struct MessageTranslationsPayload: Decodable {
         
         originalLanguage = try container.decode(String.self, forKey: .originalLanguage)
         
-        var translated = [TranslationLanguage: String]()
+        var translatedDictionary = [TranslationLanguage: String]()
```

Comment:
how about
```suggestion
        var translations = [TranslationLanguage: String]()
```

Reply:
I would agree if this was an Array ;] using the "s" - WDYT?